internal/actions/pr.go

```diff
@@ -190,19 +202,16 @@ func CreatePullRequest(ctx context.Context, repo *git.Repo, client *gh.Client, o
 	if didCreatePR {
 		action = "created"
 	} else {
-		// TODO: need to update the PR metadata possibly here
-		action = "fetched existing"
+		action = "sychronized"
 	}
 	_, _ = fmt.Fprint(os.Stderr,
-		"  - ", action, " pull request for branch ", colors.UserInput(opts.BranchName),
-		" (into branch ", colors.UserInput(prBaseBranch), "): ",
-		colors.UserInput(pull.Permalink),
-		"\n",
+		"  - ", action, " pull request ",
+		colors.UserInput(pull.Permalink), "\n",
```

Comment:
tweaked this just to be less visually noisy.
<img width="705" alt="Screen Shot 2022-09-12 at 11 34 52 AM" src="https://user-images.githubusercontent.com/773453/189730183-0007f338-2059-4169-b301-537f2263ee8f.png">